src/lib/styles/themes/dark.scss

```diff
@@ -218,4 +218,15 @@
   --negative-emphasis: var(--pink-200);
   --negative-emphasis-contrast: var(--neutral-50);
   --negative-emphasis-light: var(--pink-200-a25);
+
+  --proposal-open-background: var(--blue-accent-tint);
```

Comment:
do I understand correctly that our pattern is something like `component-state-property`?

Reply:
Good point! I found them in the new design variables in Figma. I’ll rename them accordingly.
![image](https://github.com/user-attachments/assets/4d80f7b3-64da-4dc3-8bbd-5ad9a915f71b)